haystack/document_store/base.py

```diff
@@ -19,9 +19,11 @@ class BaseDocumentStore(BaseComponent):
     index: Optional[str]
     label_index: Optional[str]
     similarity: Optional[str]
+    duplicate_documents_options: tuple = ('skip', 'overwrite', 'fail')
```

Comment:
Using Literal would be good idea but it supported from 3.8+ otherwise we need to install another dependency 
https://pydantic-docs.helpmanual.io/usage/types/#literal-type